src/clr-angular/utils/id-generator/id-generator.service.ts

```diff
@@ -10,7 +10,11 @@ let NB_INSTANCES = 0;
 
 export const UNIQUE_ID = new InjectionToken<string>("UNIQUE_ID");
 
+export function UniqueIdProvider() {
```

Comment:
should be `uniqueIdProvider`. lowercase `U`. you can fix that in your other PR. no need to update this or travis will have to run again.

Reply:
ok. will do.